style/preview.less

```diff
@@ -20,18 +20,18 @@
 
 	&-header {
 		display: flex;
-		height: 78px;
+		height: 52px;
 		line-height: 62px;
-		margin-top: 12px;
-		margin-bottom: 12px;
```

Comment:
| the header with thumbnail top margin/padding doesn't look good on | header without thumbnail looks fine | header get cutoff in smaller screen (i think it doesn't happen before) |
| ---| --- | --- |
| <img width="393" alt="image" src="https://github.com/wikimedia/wikipedia-preview/assets/2560096/7c7bd4c0-ba9e-4838-bb13-1993a34157e5">| <img width="413" alt="image" src="https://github.com/wikimedia/wikipedia-preview/assets/2560096/049228b8-50c8-4709-a2d9-0c0de40d9599"> |<img width="345" alt="image" src="https://github.com/wikimedia/wikipedia-preview/assets/2560096/fa83a172-ad33-4f06-8750-0bd63d1b3a90"> |